src/Entity/Embedded/Invite/InviteRequest.php

```diff
@@ -52,11 +52,219 @@ class InviteRequest
     private $lastName;
 
     /**
-     * @var string
+     * @var null|string
      * @Serializer\Type("string")
      */
     private $redirectUri;
 
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $declineRedirectUri;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $closeRedirectUri;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $redirectTarget;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $prefillSignatureName;
+
+    /**
+     * @var null|integer
+     * @Serializer\Type("integer")
+     */
+    private $forceNewSignature;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $signingInstructions;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $requiredPresetSignatureName;
+
+    /**
+     * @var null|string
+     * @Serializer\Type("string")
+     */
+    private $language;
+
+    /**
+     * @return string|null
+     */
+    public function getDeclineRedirectUri(): ?string
+    {
+        return $this->declineRedirectUri;
+    }
+
+    /**
+     * @param string|null $declineRedirectUri
+     *
+     * @return InviteRequest
+     */
+    public function setDeclineRedirectUri(?string $declineRedirectUri): InviteRequest
```

Comment:
What for we are allowing to send null as an attribute?

Reply:
It's not required attribute

so it can be nullable
and someone could remove it from sending request